contexts/did-v1.jsonld

```diff
@@ -42,6 +42,7 @@
       "@id": "dc:created",
       "@type": "xsd:dateTime"
     },
+    "ethereumAddress": "sec:ethereumAddress",
```

Comment:
evidence of a PR against https://github.com/w3c-ccg/security-vocab/blob/master/index.html and https://github.com/w3c-ccg/security-vocab/blob/master/contexts/security-v3-unstable.jsonld should be provided before this can be accepted.

Reply:
We need to discuss this as a group -- this is a brand name going in the DID Core context... replace it with "myspaceAddress" to understand why it's problematic. We could generalize this feature and make it "blockchainAddress" or "dltAddress" and then use something like: "blink:ethr:0x28394723948" or something to that effect.

If this is going in DID Core, we need to put it in the core spec or have a very good reason (better than the ones given thus far) to include it. Why is Ethereum getting special treatment other than "a few of us use it"... why does every other DLT have to create their own context for this?

We need to discuss this as a group.

Reply:
Ethereum is a suite of protocols and represents infrastructure in the same way as for example IPFS represents a suite of protocols.

Reply:
> Ethereum is a suite of protocols and represents infrastructure in the same way as for example IPFS represents a suite of protocols.

Sure, but that's not the point of contention. You'll notice that even though people use IPFS and DIDs together, that we don't have IPFS anything in the DID Core JSON-LD Context.

To put it another way... why is `ethereumAddress` being proposed and not `blockchainAddress`? The latter would serve more than just the Ethereum community.

Reply:
Where is the standard defintion for `blockchainAddress` if its viable, lets use it... but W3C CCG has a defintion for `ethereumAddress`... This feels a bit like moving of the goal posts... if the CCG is not reputable enough for registering properties to did core... we're going to have a really hard time because everything prefixed with `sec` is vocabulary controlled by the W3C CCG.

Reply:
You should definitely explore the work that we have been doing at CAIPs (Chain Agnostic Improvement Proposals)... so far we have standardized references to ChainIDs, AccountIDs and AssetIDs

https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-10.md

Our CAIP-10 AccountIDs would be a great fit for the blockchainAddress standard definition. Here are some examples:
```
# Ethereum mainnet
0xab16a96d359ec26a11e2c2b3d8f8b8942d5bfcdb@eip155:1

# Bitcoin mainnet 
128Lkh3S7CkDTBZ8W7BbpsN3YYizJMp8p6@bip122:000000000019d6689c085ae165831e93

# Cosmos Hub
cosmos1t2uflqwqe0fsj0shcfkrvpukewcw40yjj6hdc0@cosmos:cosmoshub-3

# Kusama network
5hmuyxw9xdgbpptgypokw4thfyoe3ryenebr381z9iaegmfy@polkadot:b0a8d493285c2df73290dfb7e61f870f
```

Reply:
@pedrouid the "sec:ethereumAddress" part is used to define a URL which provides a vocabulary defintion for what an "ethereumAddress" is.

In order to use CAIP, we would ideally define `blockchainAddress` here: https://github.com/w3c-ccg/security-vocab/pull/42

and then reference it here, as we are doing for `sec:ethereumAddress`. Would you be willing to open a PR against https://github.com/w3c-ccg/security-vocab ?

Your vocab definition can link to https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-10.md... and by doing so, anyone using the JSON-LD security vocab can make use of `blockchainAddress`.

Reply:
I would prefer `blockchainAccountId` because CAIP-10 also refers to it as `accountId`.

Reply:
yes, that makes sense.

Reply:
Agreed, `blockchainAccountId` is more accurate

Reply:
Is `blockchainAccountId` only intended to be specific to Ethereum? I ask because many other ledger systems don't use an account paradigm.

Reply:
its based on the CAIP-10 defintion, see https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-10.md#test-cases

Reply:
I would suggest `ledgerAddress`, as that is the most ledger/chain agnostic term I can think of, given what we're defining here. Does anyone feel strongly that we must introduce the Account concept into this, given that actually means something specific within certain target systems, for which we want this to be a general reference for?

Reply:
-1 to ledger address (its not defined anywhere)... I prefer we use `ethereumAddress` and then work to leverage CAIP or some other existing standard to get support for all the other ledger addresses.

Reply:
+1 for using LedgerAddress, it is more generic and removes the confusion between account based vs. UTXO based blockcahin